java_gen/templates/custom/OFGroupDeleteVer.Builder.java

```diff
@@ -0,0 +1,20 @@
+
+@Override
```

Comment:
I don't think you need to manually implement these methods that throw `UnsupportedOperationException`. If you just don't provide a custom template, loxigen *should* generate them automatically?